pandas/io/excel/_xlwt.py

```diff
@@ -63,6 +63,11 @@ def __init__(
         self.fm_datetime = xlwt.easyxf(num_format_str=self.datetime_format)
         self.fm_date = xlwt.easyxf(num_format_str=self.date_format)
 
+    @property
+    def sheets(self) -> dict[str, Any]:
+        result = {sheet.name: sheet for sheet in self.book._Workbook__worksheets}
```

Comment:
Could directly return the dict like above

Reply:
In general I like having the return line separate as it gives a variable to inspect allowing for easier debugging. Actually meant to do this above. I'll change the above for now, but if others don't like this style I have no objection to having the return all on one line.

Reply:
APart from consistency no strong opinion on my side